packages/application-shell/src/components/fetch-project/fetch-project.graphql

```diff
@@ -31,8 +31,12 @@ query ProjectQuery($projectKey: String!) {
       canAddCustomers
       canAddCustomerGroups
       canAddDiscountCodes
+      canAddProductDiscounts
+      canAddCartDiscounts
```

Comment:
So apparently those two are wrong since they are not in our BE schema 
![image](https://user-images.githubusercontent.com/11442895/49864470-d4e57f80-fe02-11e8-81a0-4a9e6484a5ac.png)

cc @tdeekens

Reply:
Is this a leftlover? Should I create quick PR in the BE for adding them? 🤔

Reply:
Feel free to yes.